src/react_native/core.cljs

```diff
@@ -67,6 +67,10 @@
 
 (def status-bar (.-StatusBar ^js react-native))
 
+(defn set-status-bar-style
+  [style]
+  (.setBarStyle (.-StatusBar ^js react-native) style true))
```

Comment:
we shouldn't use it because it will conflict with navigation, status-bar should be managed from screen properties

Reply:
@flexsurfer Oh I see, so how to do it please? I have tried `:options {:statusBar {:style :light}}` but it is not working

Reply:
you need to set background color as well

working example
```
:options   {:topBar        {:visible false}
                :navigationBar {:backgroundColor colors/black-persist}
                :statusBar     {:backgroundColor colors/black-persist
                                :style           :light}}
                                ```

Reply:
@flexsurfer I have tried this. It does not work.

Reply:
then we need to find out why, and fix it , because it should work